data/sql/updates/pending_db_world/update-sai-ssc-trash.sql

```diff
@@ -0,0 +1,120 @@
+(21966, 0, 6, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 38371, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - In Combat - Cast Bestial Wrath'),-- linked but %hp based? missing ability 38449 as well?
+(21966, 0, 7, 8, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - On Aggro - Set In Combat With Zone'),
+(21966, 0, 8, 0, 61, 0, 100, 0, 0, 0, 0, 0, 0, 0, 39, 10, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - On Aggro - Set In Combat With Zone'),
+(21966, 0, 9, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 38455, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Fathom-Guard Sharkkis - On Death - Cast Power of Sharkkis'),
+(22009, 0, 0, 0, 60, 0, 100, 1, 100, 100, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Tainted Elemental - On Reset - Set In Combat With Zone'),
+(22009, 0, 1, 0, 60, 0, 100, 0, 100, 500, 2350, 2650, 0, 0, 11, 38253, 0, 0, 0, 0, 0, 5, 200, 0, 0, 0, 0, 0, 0, 0, 'Tainted Elemental - In Combat - Cast Poison Bolt'),-- fully sniffed
+(22009, 0, 2, 0, 60, 0, 100, 769, 15000, 15000, 0, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Tainted Elemental - On Update - Despawn'),
+(22055, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Elite - On Aggro - Set In Combat With Zone'),
+(22055, 0, 1, 0, 0, 0, 100, 0, 6850, 17250, 1250, 15750, 0, 0, 11, 38260, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Elite - In Combat - Cast Cleave'),-- fully sniffed
+(22055, 0, 2, 0, 0, 0, 100, 0, 15350, 17750, 8000, 10000, 0, 0, 11, 38262, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Elite - In Combat - Cast Hamstring'),-- repeat not sniffed
+(22055, 0, 3, 0, 34, 0, 100, 0, 8, 1, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Elite - Movement Inform - Set In Combat With Zone'),
+(22056, 0, 0, 0, 4, 0, 100, 0, 0, 0, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Strider - On Aggro - Set In Combat With Zone'),
+(22056, 0, 1, 0, 0, 0, 100, 0, 15700, 15700, 10850, 13350, 0, 0, 11, 38259, 0, 0, 0, 0, 0, 5, 40, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Strider - In Combat - Cast Mind Blast'),-- fully sniffed
+(22056, 0, 2, 0, 34, 0, 100, 0, 8, 1, 0, 0, 0, 0, 38, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Coilfang Strider - Movement Inform - Set In Combat With Zone'),
+(22250, 0, 0, 0, 25, 0, 100, 769, 0, 0, 0, 0, 0, 0, 41, 21000, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Rancid Mushroom - On Reset - Despawn'),
+(22250, 0, 1, 0, 0, 0, 100, 0, 1150, 1150, 1200, 3400, 0, 0, 11, 31698, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Rancid Mushroom - In Combat - Cast Grow'),-- fully sniffed
+(22250, 0, 2, 0, 0, 0, 100, 0, 22950, 22950, 19400, 23400, 0, 0, 11, 38652, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Rancid Mushroom - In Combat - Cast Spore Cloud');-- fully sniffed
```

Comment:
Better to check directly in the parsed txt for this one

Reply:
Initial timers/repeat timers for grow are similar to mangos value. theirs seem unsniffed, sporecloud happens on death according to them but the initial timers above seem correct